app.json

```diff
@@ -5,6 +5,7 @@
       "buildpacks": [
         { "url": "heroku/nodejs" },
         { "url": "https://github.com/heroku/heroku-buildpack-google-chrome" },
+        { "url": "https://github.com/heroku/heroku-buildpack-chromedriver" },
```

Comment:
Do we need the app.json now? We probably don't need to build in Heroku as well as Azure.

Reply:
I figured I'd leave both there for the time being until we decide to switch then remove traces of the Heroku one separately